service/src/main/resources/api/service_openapi.yaml

```diff
@@ -1923,6 +1923,66 @@ components:
         - operator
         - excluded
 
+    CriteriaGroupCollectionV3:
```

Comment:
I don't love this, though I don't hate it either. The double group-collection is a bit awkward. Some other ideas:

- CriteriaBlock/CriteriaGroupBlock
- CriteriaSection/CriteriaGroupSection <-- maybe my favorite?

- EntityFilterGroup
  +- EntityFilter
    +- Criteria

Reply:
I'd like to not reuse "EntityFilter" for this object, to avoid confusion with defining a query. This is more of a UI object, at least for now while we're building the filter on the front end. "Section" SGTM.